core/modules/link/tests/link.custom.test

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @file
+ * Basic simpletests to test basic link fields in a custom form.
+ */
+
+require_once(__DIR__ . '/link.test');
+
+/**
+ * Testing that the basics work for link fields on custom forms.
+ */
+class LinkCustomFormTest extends LinkBaseTestClass {
+  /**
+   * Test the output of a custom form with link fields.
+   */
+  public function testCustomLinkFields() {
+    $group = 'Link field custom form';
+    $this->backdropGet('link-field-custom-form');
+    $this->assertPattern('@<label for="edit-link-test-defaults-title">Title@is', "Defaults", $group);
```

Comment:
Using `@` as regex delimiters is unusual. Could these be replaced with the typical slashes? Otherwise, using `!` is the second-most common delimiter.